opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/resources/mockedsandbox/restrecord/redirect/accounts/sandbox/mappings/mapping-v1-consents-24281.json

```diff
@@ -8,7 +8,7 @@
         "matches" : "[0-9a-zA-z\\-]+"
       },
       "PSU-ID" : {
-        "equalTo" : "anton.brueckner"
+        "matches" : "[a-z\\.]*"
```

Comment:
The expectation should be clear if you want to use no PSU ID - you need to create a new fixture package